onelogin/get.go

```diff
@@ -253,6 +304,28 @@ func getDevice(devices []Device) (device *Device, err error) {
 		return
 	}
 
+	if opts.MfaDevice != "" {
+		for _, d := range devices {
+			if d.DeviceType == opts.MfaDevice {
+				device = &d
+				fmt.Printf("MFA device %s found, automatically selecting it.\n", opts.MfaDevice)
```

Comment:
These print statements should be adopted to the new logging mechanism.

Reply:
Moved to log statements